packages/webpack-cli/lib/utils/prompt-installation.js

```diff
@@ -51,7 +51,7 @@ async function promptInstallation(packageName, preMessage) {
             process.exit(2);
         }
 
-        return utils.packageExists(packageName);
+        return packageName;
```

Comment:
No need to check package on exists, because if it was failed we will exit with `2` code above